test/node/resendable-call-from-client.js

```diff
@@ -44,6 +44,8 @@ function createServer() {
         sessionStorage.set(args[0], args[1]);
         if (initialConnection) {
```

Comment:
Why don't just `initialConnection ? call() : close();` I'd prefer single line for such simple cases or at least
```js
if (initialConnection) call();
else close();
```

Reply:
In addition, we can use `handlers[message]` instead of `switch/case`

Reply:
Using `handlers` is a matter for another PR.
IMHO if you use else you must use brackets. As per ternary expression, this PR is already hard to fully understand because of the master-child communication process, so I would like to leave the rest of the code as simple as it can be. If you insist on using the ternary expression, I will rewrite it, but I would prefer not to.

Reply:
Using `handlers` is a matter for another PR. IMHO if you use else you must use brackets. As per ternary expression, this PR is already hard to fully understand because of the master-child communication process, so I would like to leave the rest of the code as simple as it can be. If you insist on using the ternary expression, I will rewrite it, but I would prefer not to.

Reply:
Ok